helm/cluster-cloud-director/values.schema.json

```diff
@@ -129,46 +146,62 @@
                 "ovdcNetwork",
                 "site"
             ],
+            "title": "Cloud Director",
             "type": "object"
         },
         "cloudProvider": {
             "properties": {
                 "enableVirtualServiceSharedIP": {
+                    "description": "If enabled, multiple virtual services can share the same virtual IP address.",
+                    "title": "Enable sharing of IPs in virtual services",
                     "type": "boolean"
                 },
                 "oneArm": {
+                    "description": "TODO.",
                     "properties": {
                         "enabled": {
+                            "title": "Enable",
                             "type": "boolean"
                         }
-                    }
+                    },
+                    "title": "TODO"
                 }
             },
             "required": [
                 "enableVirtualServiceSharedIP",
                 "oneArm"
             ],
+            "title": "TODO",
             "type": "object"
         },
         "cluster": {
             "properties": {
                 "parentUid": {
+                    "description": "If set, create the cluster from a specific management cluster associated with this UID.",
+                    "title": "Management cluster UID",
                     "type": "string"
                 },
                 "rdeId": {
+                    "description": "TODO",
```

Comment:
What is the `rdeId`?

Reply:
This is the cluster entity ID in the VCD API.